pages/pricing.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react'
+import {
+	AiOutlineCheckCircle
+} from 'react-icons/ai'
+import Navbar from '../views/Navbar';
+import Footer from '../components/Footer';
+import Link from 'next/link';
+
+const pricingPlan = [
+	{
+		pricingName: 'Free',
+		duration: '.',
+		pricing: 'for open source projects',
+		features: ['Access of all features', 'Unlimited team size', 'For both Github and Bitbucket'],
+		buttonText: 'Add a Repo',
+		link: 'https://github.com/'
+
+	},
+	{
+		pricingName: 'Standard',
+		duration: 'per user/month',
+		pricing: '$100',
+		features: ['Access of all features', 'personal team support for setting up'],
+		buttonText: 'Start your 30/day trial',
+		link: 'https://chrome.google.com/webstore/detail/vibinex/jafgelpkkkopeaefadkdjcmnicgpcncc' // temp. adding chrome extension link, need to remove it with payment link
```

Comment:
We can discuss on whether we should redirect them to the Chrome Extension or the login page.
And anyway, do you think we should start with a payment link even after we integrate a payment portal?

Reply:
Since the payment link is not there, that's why I added extension link.